utils/build/docker/cpp_nginx/nginx/app.sh

```diff
@@ -3,7 +3,7 @@
 backend &
 
 if [[ "${DDPROF_ENABLE:-,,}" == "yes" ]]; then
-  ddprof -l notice nginx -g 'daemon off;'
+  ddprof -l notice nginx-debug -g 'daemon off;'
```

Comment:
it would be better to have the default to be using the prod binaries. Though, there is an option that activate debug logs (`--force-dd-trace-debug`), this option sets the env var `DD_TRACE_DEBUG=true`. What about using `nginx-debug` if `DD_TRACE_DEBUG=true`, and use `nginx` otherwise ? 

Note that if you add `SYSTEM_TESTS_FORCE_DD_TRACE_DEBUG=true` to your `.env`, you'll always have this activated locally (more info here : https://github.com/DataDog/system-tests/blob/main/docs/execute/dd-trace-debug.md)